larpix/configuration/configuration.py

```diff
@@ -0,0 +1,183 @@
+from bitarray import bitarray
+import os
+import errno
+import functools
+from collections import OrderedDict
+
+from .. import bitarrayhelper as bah
+from .. import configs
+
```

Comment:
Using ``__all__`` allows ``from configuration import *`` to import ``_Smart_List`` too even though it's "private"

```suggestion

__all__ = [
    'BaseConfiguration',
    '_Smart_List',
]
```